tests/day_3.py

```diff
@@ -0,0 +1,17 @@
+from aoc import day_3
+import pytest
+
+test_lines = """vJrwpWtwJgWrhcsFMMfFFhFp
+jqHRNqRjqzjGDLGLrsFMfFZSrLrFZsSL
+PmmdzqPrVvPwwTWBwg
+wMqvLMZHhHMvwLHjbvcjnnSBnvTQFn
+ttgJtRGJQctTZtZT
+CrZsJsPPZsGzwwsLwLmpwMDw"""
+
+def test__error_sum():
+    test_rucksacks = test_lines.split('\n')
```

Comment:
minor nit, you could just throw the `.split('\n')` on the string literal above on line 9, then call `test_lines` > `test_rucksacks`, and omit this line and line 16.